src/resource_clients/store_collection.ts

```diff
@@ -0,0 +1,69 @@
+import ow from 'ow';
+import { ApiClientSubResourceOptions } from '../base/api_client';
+import { ResourceCollectionClient } from '../base/resource_collection_client';
+import { PaginatedList } from '../utils';
+import { ActorStats } from './actor';
+
+/**
+ * @hideconstructor
+ */
+export class StoreCollectionClient extends ResourceCollectionClient {
+    constructor(options: ApiClientSubResourceOptions) {
+        super({
+            resourcePath: 'store',
+            ...options,
+        });
+    }
+
+    /**
+     * https://docs.apify.com/api/v2/#/reference/store/store-actors-collection/get-list-of-actors-in-store
+     * @param {object} [options]
+     * @param {number} [options.limit]
+     * @param {number} [options.offset]
+     * @param {string} [options.search]
+     * @param {string} [options.sortBy]
+     * @param {string} [options.category]
+     * @param {string} [options.username]
+     * @param {string} [options.pricingModel]
+     * @return {Promise<PaginationList>}
```

Comment:
i'd remove this, it adds no value, the information is all available on type level (and it wont have any effect on the docs either)